tools/hive-bigquery/hive_to_bigquery.py

```diff
@@ -0,0 +1,257 @@
+"""Main Module to migrate Hive tables to BigQuery"""
+
+import logging
+
+from bigquery_component import BigQueryComponent
+from bigquery_table import BigQueryTable
+import custom_exceptions
+from gcs_storage_component import GCSStorageComponent
+from hive_component import HiveComponent
+from hive_table import HiveTable
+from mysql_component import MySQLComponent
+from properties_reader import PropertiesReader
+from resource_validator import ResourceValidator
+import init_script
+from utilities import print_and_log
+
+logger = logging.getLogger('Hive2BigQuery')
+
+
+def compare_rows(bq_component, hive_component, gcs_component, hive_table_model,

```

Comment:
Rename to `compare_row_counts` to clarify you are not validating that the rows match.